src/configs/strings.ts

```diff
@@ -1,3 +1,4 @@
 export const GLOBAL_STRINGS = {
   unassign_comment: "Releasing the bounty back to dev pool because the allocated duration already ended!",
+  ask_update: "Do you have any updates?",
```

Comment:
We should be using camelCase. Only underscores for CONSTANT_VARIABLES

Reply:
This doesn't look like you changed it so you shouldn't mark it as resolved

Reply:
https://github.com/ubiquity/bounty-bot/pull/105/commits/eb8abcaa7a839addd4e7a1c2872149745ccb809f

can I make it resolved? lol